### DISCUSSION TITLE
Unable to Create Project on WalletConnect Cloud

### DISCUSSION THREAD

**Discussion by TheDARKFURY** [WalletConnect/walletconnect-monorepo#4183] [Developer Support]
I am trying to create a project using WalletConnect Cloud and I am getting the error - `insert or update on table "projects" violates foreign key constraint "projects_creator_fkey"`

![wallet connect error](https://github.com/WalletConnect/walletconnect-monorepo/assets/62480336/7d3e1335-9f1d-4119-a989-6ce7b38b96e9)

**Comment by glitch-txs** [comment]
hi are you still having this issue?

**Reply by TheDARKFURY** [reply]
No, the issue has been resolved.

**Comment by montajebi** [comment]
I have the same issue!

**Comment by gordo-labs** [comment]
Same issue here
